lib/game_stats.rb

```diff
@@ -1,4 +1,94 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score(season) # integer
+		if away_winners(season) > home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) < home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def lowest_total_score(season)
+		if away_winners(season) < home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) > home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def get_games(season) # helper
+		@all_games = @games
+	end
+
+	def home_winners(season) # helper
+		@home_wins = []	
+			@games.each do |game|
+				if game.away_goals.to_i < game.home_goals.to_i	
+					@home_wins << game.home_goals.to_i
+				end
+			end
+		@home_wins.sum
+	end
+
+	def away_winners(season) # helper
+		@away_wins = []
+			@games.each do |game|
+				if game.away_goals.to_i > game.home_goals.to_i	
+					@away_wins << game.away_goals.to_i
+				end
+			end
+		@away_wins.sum
+	end	
+
+	def percentage_home_wins(season) # float
+		home_winners(season)
```

Comment:
same thing here as above - I don't think we want season as a parameter. We should be looking at the entire data set and determining what percentage of all games was home wins, what percentage was away wins, and what percentage was ties